### ISSUE TITLE
Grand Unified Hit Location Table

### ISSUE THREAD

**Issue by Dvorn1543** [richardwilkes/gcs_library#133]
Now that space is used a bit differently in the newest version of GCS, the hit location column has a decent bit extra space. Would it be possible to add the hit locations introduced in MA and LT, and compiled together in the Grand Unified Hit Location Table, to the software?

**Comment by richardwilkes** [comment]
Probably not in the general case. That extra space collapses very quickly when you have characters with super Strength.

That said, it has been a long time since I've looked at the more detailed hit location tables. Are they flexible enough to support all of the body types? How many extra entries are there altogether?

**Comment by Dvorn1543** [comment]
Yup, just about every body type would have em, assuming the body type actually has the limb(s) in question. 

Effectively these are things like: Forearm, Shin, Elbow, Shoulder, Abdomen and Chest(As opposed to just Torso) etc.

It would allow for loadouts which do not have full limb/torso coverage to be easily represented in the hit location box, as opposed to being represented via extensive notes in the equipment section.

All the LOCATIONS(Component Parts):

HEAD(Skull, Face)
NECK
TORSO(Chest, Abdomen, Groin)
ARMS(Shoulders, Upper Arms, Elbows, Forearms)
HANDS
LEGS(Thighs, Knees, Shins)
FEET
